packages/network/clients/chronobankNodes/bitcoin.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import axios from 'axios'
+
+const timeout = 10000
+
+export const middleware_bitcoin_mainnet_rest = {
+  client: axios.create({
+    baseURL: 'https://middleware-bitcoin-mainnet-rest.chronobank.io',
+    responseType: 'json',
+    timeout,
+  }),
+  options: {
+    interceptors: {
+      response: [{
+        // eslint-disable-next-line no-unused-vars
+        success: ({ getState, dispatch, getSourceAction }, request) => {
```

Comment:
Could we delete such params to remove eslint errors?

Reply:
But ESLint disabled for lines like this in  the comment above.
If we will remove these unused arguments, then we will need to search info about accepted arguments of axios's success method again. I kept them for information. 

Probably, in the great future, this interceptor may be enhanced and these args may be used.